src/Icons/composer.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "symfony/ux-icons",
+    "type": "symfony-bundle",
+    "description": "Twig component to use svg icons in your Symfony app",
```

Comment:
I think almost all Symfony component have been reworded in a not-so-distant past, and the "rule" seems to me to say what it "does" more than what it "is"..

https://packagist.org/search.json?q=symfony/

I wrote that for mine: 

> Renders and transforms SVG icons, manages local or remote Icon Packs and generates SVG sprites.

 but as we have trimmed down, we should ... trim the description too :))

Reply:
"Renders local and remote SVG icons in your Twig templates." ?